presto-spi/src/main/java/com/facebook/presto/spi/resourceGroups/ResourceGroupInfo.java

```diff
@@ -142,7 +156,6 @@ public ResourceGroupInfo createSingleNodeInfo()
                 getNumEligibleSubGroups(),
                 getMemoryUsage(),
                 getNumAggregatedRunningQueries(),
-                getNumAggregatedQueuedQueries(),
-                emptyList());
+                getNumAggregatedQueuedQueries());
```

Comment:
This is an unrelated change, and adds very little. There is only one invocation of this function.